src/testutils.jl

```diff
@@ -1,7 +1,10 @@
 # Utilities to support the testing of distributions and samplers
 
-import Test: @test
-
+if VERSION >= v"0.7.0-DEV"
```

Comment:
This check isn't quite specific enough, since earlier versions of 0.7 still had Base.Test. I'd recommend just doing
```julia
using Compat
using Compat.Test
```
in each of the relevant files. This requires adding an entry for Compat in test/REQUIRE if no such entry exists, and the minimum version should be 0.34.0 (I think).

Reply:
I put forward another view in that if people are using 0.7.0-DEV there is no reason to be using earlier versions of it as it's purely a nightly build version. They should just switch.

Reply:
It's not feasible for everyone to upgrade often. Making the check more specific is as easy as replacing `v"0.7.0-DEV"` in what you have here with `v"0.7.0-DEV.2004"`.

Reply:
Sorry, but the well-established rule is that we should either find the specific commit which introduced the change, or use `isdefined` to check for the change directly.  This is useful e.g. if one needs to test an earlier development version of Julia to check for a regression.

Anyway, when there's support in Compat, it should be used instead of explicit `VERSION` checks.